app/main/plugins/core/basic-apps/linux.js

```diff
@@ -0,0 +1,106 @@
+import { remote } from 'electron'
+import path from 'path'
+import fs from 'fs'
+import uniq from 'lodash/uniq'
+import flatten from 'lodash/flatten'
+import { shellCommand } from 'cerebro-tools'
+
+let appDirs = [
+  path.join(remote.app.getPath('home'), '.local', 'share'),
+  path.join('/usr', 'share'),
+  path.join('/usr', 'share', 'ubuntu'),
+  path.join('/usr', 'share', 'gnome'),
+  path.join('/usr', 'local', 'share'),
+  path.join('/var', 'lib', 'snapd', 'desktop')
+]
+
+if (!!process.env.XDG_DATA_DIRS) {
+  appDirs = [
+    ...appDirs,
+    ...process.env.XDG_DATA_DIRS.split(':')
+  ]
+}
+
+// Icon resolutions in priority of checking
+const iconResolutions = [
+  '128x128',
+  '96x96',
+  '64x64',
+  '256x256',
+  '512x512'
```

Comment:
out of curiosity, why not always go from high-res to low-res? why 64x64 has higher priority than 256x256?